src/angular-phrase.ts

```diff
@@ -76,7 +76,7 @@ phrase.directive("phraseJavascript", ["phraseEnabled", "phraseProjectId", "phras
             replace: true,
             link: (): void => {
                 if (phraseEnabled) {
-                    const url = `https://phraseapp.com/assets/in-context-editor/2.0/app.js?${new Date().getTime()}`;
+                    const url = `https://app.phrase.com/assets/in-context-editor/2.0/app.js?${new Date().getTime()}`;
```

Comment:
Hey @iamandrewluca! Thanks for the contribution and nice catch!
Can you please change the domain to `https://phrase.com` instead?
```suggestion
                    const url = `https://phrase.com/assets/in-context-editor/2.0/app.js?${new Date().getTime()}`;
```

Reply:
Nice, didn't know assets are also served from main domain.